docker/lnd/start-lnd-btc.sh

```diff
@@ -51,4 +47,13 @@ if [[ "$NODE" == "bitcoind" ]]; then
     PARAMS="$PARAMS --bitcoind.zmqpubrawtx=$ZMQPUBRAWTX"
 fi
 
+# We want to make it easy for devs to test functionality of the engine which could
+# potentially involve the constant restarting of a particular daemon. It can get
+# annoying to continually have to unlock/create wallets so we will enable noseedbackup
+# ONLY if the network is outside of mainnet AND dev is set to true
```

Comment:
your comment mentions that this only happens if dev is set to true, but the conditional doesn't check that. Which one is wrong?

Reply:
the comment is wrong. I changed the name from DEV to `NO_SEED_BACKUP`